collect_app/src/androidTest/java/org/odk/collect/android/instrumented/forms/FormNavigationTest.java

```diff
@@ -65,7 +65,7 @@
             .around(new RunnableRule(() -> {
                 // Set up demo project
                 AppDependencyComponent component = DaggerUtils.getComponent(ApplicationProvider.<Application>getApplicationContext());
-                component.projectImporter().importDemoProject();
+                component.projectImporter().importNewProject(Project.Companion.getDEMO_PROJECT());
```

Comment:
For some reason I couldn't get the companion field to be static (even with `@JvmStatic`)